### ISSUE TITLE
Ghazghkull thrakka has a new model since apoc release

### ISSUE THREAD

**Issue by kolslaw68** [normanthesquid/wh40k-fundapocalypse#21]
**Faction:** Orks

**Unit:** Ghazghkull thrakka

**Issue Type:** Incorrect Datasheet

**Description:** Proposed new sheet:

Ghazghkull Thraka [10 PL]: Gork's claw, Mork's roar
Abilities: Great Waaagh!, Waaagh! (Goff), Grand Warboss, Unit: Ghazghkull Thraka, Weapons: Gork's claw, Mork's roar
Profiles:
Great Waaagh!: Description:Add 1 to the Attacks characteristic of friendly ORKS LIGHT units whilst they are making Fight actions whilst within 6" of this unit.|Warhammer 40,000: Apocalypse
Waaagh! (Goff): Description:Re-roll hit rolls of 1 for attacks made with melee weapons by friendly GOFF units whilst they are within 6" of this unit.|Warhammer 40,000: Apocalypse
Grand Warboss: This model can be included in an ork detachment without preventing other units from gaining a clan kultur.  This model does not benefit from any clan kultur unless the clan kultur selected for its detachment is the Goffs clan kultur.
Ghazghkull Thraka: M:5"|WS:2+|BS:5+|A:2|W:2|Ld:6|Sv:3+|Warhammer 40,000: Apocalypse
Kustom Klaw: Type:Melee|Range:Melee|A:User|SAP:7+|SAT:6+|Abilities:-|Warhammer 40,000: Apocalypse
Twin Big Shoota: Type:Heavy|Range:36"|A:3|SAP:7+|SAT:9+|Abilities:-|Warhammer 40,000: Apocalypse

**Comment by kolslaw68** [comment]
In psychic awakening he gained 4 wounds and the ability to only suffer 4 wounds a phase.  His melee attack gained 1 ap and 1 damage, and his gun doubled the number of shots.  his PL went up by 2
[6:08 PM]
I also added language that he wouldn't break clans even if he wan't in a goff detachment, which he gained in psychic awakening.  I just copied the text from his new datasheet and don't know if that is relevant to apoc
